component/firecracker-base/orchestrate-install.sh

```diff
@@ -198,18 +205,34 @@ execute_cleanup() {
         *)
             echo "Error: Something went wrong during cleanup, OS_VARIANT set to: $OS_VARIANT" && exit 1
         ;;
-    esac  
+    esac
 
     rm -Rf /tmp/firecracker-install/*
-    
+
+}
+
+prepare_jailers() {
+  if test -f "./prepare_jailer.sh"; then
+    ITERATIONS="${1:-100}" # Default to 100 jails
+    echo "Creating $ITERATIONS jails..."
+    for (( iter=0; iter<$ITERATIONS; iter++ ))
+    do
+        ./prepare_jailer.sh $iter &
+    done
+    wait
+  else
+    echo "prepare_jailer.sh script not found, skipping jail creation."
```

Comment:
nit: I think it would be preferable to exit non-zero here so we stop

Reply:
My counterpoint is that we do cleanup after this step and I imagine we still want that to execute. Ignore the fact that our cleanup doesn't do anything on ubuntu...

Reply:
I'm not sure, I would see this is a "fatal" as the platform wasn't ready for firecracker'ing. Happy to have it either way, we don't really have a "standard" or a cleanup process at all